### PR TITLE
updated twitter logo

### PR DESCRIPTION
**Pull request by spellbind0127**

### CODE REVIEW

File: img/t/twitter.com.svg [hunk 1]
```diff
@@ -1 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" class="r-k200y r-13gxpu9 r-4qtqp9 r-yyyyoo r-np7d94 r-dnmrzs r-bnwqim r-1plcrui r-lrvibr" width="22.706" height="22.706"><path d="M22.706 4.311c-.835.37-1.732.62-2.675.733a4.67 4.67 0 002.048-2.578 9.3 9.3 0 01-2.958 1.13 4.66 4.66 0 00-7.938 4.25 13.229 13.229 0 01-9.602-4.868c-.4.69-.63 1.49-.63 2.342a4.66 4.66 0 002.072 3.878 4.647 4.647 0 01-2.11-.583v.06a4.66 4.66 0 003.737 4.568 4.692 4.692 0 01-2.104.08 4.661 4.661 0 004.352 3.234 9.348 9.348 0 01-5.786 1.995A9.5 9.5 0 010 18.487a13.175 13.175 0 007.14 2.093c8.57 0 13.255-7.098 13.255-13.254 0-.2-.005-.402-.014-.602a9.47 9.47 0 002.323-2.41z" fill="#1da1f2"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none" class="u01b__icon-home">
+    <path d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path>
+</svg>
```
Comment (hkamran80) [inline img/t/twitter.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 1227"><path d="M714 519 1161 0h-106L667 451 357 0H0l468 682L0 1226h106l409-476 328 476h357L714 519ZM569 688l-47-68L144 80h163l304 436 48 68 396 566H892L569 688Z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
I like the name "Twitter" and the blue logo way better.

Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Comment by spellbind0127** [comment]
@hkamran80 for some reason it seems like the twitter logo isn't being updated on https://2fa.directory is this an issue or is it just me

**Comment by Carlgo11** [comment]
@spellbind0127 images are heavily cached so that the website loads faster for everyone. Clearing your browser cache will most likely update the logo.

**Comment by spellbind0127** [comment]
Yah seems like the page only updates when I complain it’s updated now.

**Pull request merged**